### DISCUSSION TITLE
Verify that Grid Virtualization is Enabled

### DISCUSSION THREAD

**Discussion by SkylerLutz** [IgniteUI/igniteui-angular#13901] [Q&A] [answered]
Hi, like the title says, how can I Verify that Grid Virtualization is Enabled? The docs imply that it “just works” out of the box (I do not set grid width and height) but I am seeing poor scroll performance (lots of change detection, scroll listeners, layout recalcs) and I want to rule out virtualization as a contributing factor. Thanks in advance.

**Comment by IvanKitanov17** [comment, marked as answer]
Hi @SkylerLutz,  the description in [this section](https://www.infragistics.com/products/ignite-ui-angular/angular/components/grid/virtualization#enabling-virtualization) of the “Virtualization and performance” topic in our documentation, outlines the internal usage of the igxForOf directive by the IgxGrid. This means that the grid already implements virtualization for its records as long as you do not specify height or width for the grid or if you explicitly set them to null.

If you would like to observe the performance of the grid without virtualization, simply set the height and the width of the grid to a specific value and see if there is difference in the performance.
